frog/imports/api/graphs.js

```diff
@@ -3,8 +3,16 @@ import { uuid } from 'frog-utils'
  
 export const Graphs = new Mongo.Collection('graphs');
 
-export const addActivity = () =>
+export const addGraph = () => {
+  const id = uuid()
   Graphs.insert({
-    _id: uuid(),
-    created_at: new Date() 
+    _id: id,
+    name: 'untitled',
```

Comment:
might be cleaner to use a default argument

`addGraph = (name = 'untitled') => {}`

that way we can use the function in the future if we decide to create a new graph that already has a name etc.